druid/src/event.rs

```diff
@@ -84,6 +84,11 @@ pub enum Event {
     ///
     /// [`set_cursor`]: struct.EventCtx.html#method.set_cursor
     MouseMoved(MouseEvent),
+    /// Called when the mouse has left the application area.
+    ///
+    /// The `MouseLeave` event is propagated to the active widget, if
+    /// there is one, otherwise to hot widgets (see `HotChanged`).
+    MouseLeave,
```

Comment:
*~the application area~ the window*

Also the event will be propagated to both active and hot widgets. Multiple can be active, and hot widgets will receive this regardless of whether there are active.

Reply:
It's true, I copied the comment and behavior from `MouseMove`, but I think they don't match there either?

Reply:
Probably remnants of an era where druid was single window or something.